packages/custom-functions-metadata/test/cases/error-linkedentitydataprovider-excludefromautocomplete/functions.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+/**
+ * Test the linkedEntityDataProvider tag in combination with the excludeFromAutoComplete tag
+ * @param {unknown} linkedEntityId Unique `LinkedEntityId` of the `LinkedEntityCellValue`s which is being requested for resolution/refresh.
```

Comment:
In TypeScript, the @param shouldn't have the type in curly braces. These come from the actual function param type. For JavaScript where you cannot specify type info in code, the curly braces are where type info is expressed instead. Please fix this up in the test cases. Thanks!